src/backend/src/prisma/schema.prisma

```diff
@@ -735,14 +735,23 @@ model Design_Review {
   notificationSlackThreads Message_Info[]
 }
 
+model Availability {
+  availabilityId     String            @id @default(uuid())
+  scheduleSettingsId String
+  scheduleSettings   Schedule_Settings @relation(fields: [scheduleSettingsId], references: [drScheduleSettingsId])
+
+  // Availibilies are integers between 0 and 83 from 10am - 10pm (Monday - Sunday) see meetingTime field in Design_Review
+  availability Int[]
+  dateSet      DateTime @default(now())
```

Comment:
Is there a specific reason this is named dateSet? I feel like weekDate or smthn would be more accurate

Reply:
yeah i wanna keep it more vague for when we dont just have it week by week but ideally have it the next seven days from the day you select